snaps/learn/tutorials/gas-estimation.md

```diff
@@ -128,9 +128,9 @@ Edit the `files` array and add the `images/` folder:
 
 ### 3. Enable network access
 
-To enable your Snap to use the `fetch` API, enable the
-[`endowment:network-access`](../../reference/permissions.md#endowmentnetwork-access) 
-permission by adding `"endowment:network-access": {}` to the `initialPermissions` object in `snap.manifest.json`:
+To enable your Snap to [access the internet using the `fetch` API](../../features/network-access.md),
+request the [`endowment:network-access`](../../reference/permissions.md#endowmentnetwork-access) 
+permission in `packages/snap/snap.manifest.json`:
```

Comment:
Not sure about this change. `packages/snap` assumes the template Snap, which may not always be the case. I feel like it's better to just mention `snap.manifest.json`.

Reply:
This change is specifically for [this tutorial](https://docs.metamask.io/snaps/learn/tutorials/gas-estimation/), which uses the template generated by `create-snap`, so being specific with the file location should be fine (and consistent with the rest of the tutorial).

Reply:
Ah, right. Looks good to me in that case!